src/js/components/Data/DataForm.js

```diff
@@ -289,9 +291,9 @@ export const DataForm = ({
         {footer !== false && updateOn === 'submit' && (
           <Footer
             flex={false}
-            margin={{ top: 'medium' }}
             pad={{ horizontal: pad, bottom: pad }}
-            gap="small"
+            margin={theme.dataFilters?.footer?.actions?.margin}
```

Comment:
Thought about this more after our discussion and I think the dataFilters naming makes sense even though this is in DataForm

Reply:
thank you!